### PR TITLE
Adding `get-manifests` command

### PR DESCRIPTION
**Pull request by ejether**

### CODE REVIEW

**Comment by ejether** [comment]
This is branched from https://github.com/FairwindsOps/reckoner/pull/210

So should not be merged until that is.

File: reckoner/helm/provider.py [hunk 1]
```diff
@@ -42,7 +42,12 @@ def execute(cls, helm_command):
 
         # start by creating a command line arguments list with the command being first
         args = list([instance._helm_binary])
-        args.append(instance._helm_command.command)
+
+        # if command has a space in it (like get manifests), split on space
+        # and append each segment as it's own list item to make `call` happy
+        for command_segment in instance._helm_command.command.split(' '):
+            args.append(command_segment)
+
```
Comment (ejether) [inline reckoner/helm/provider.py:50]: This is a potentially risky change.

**Comment by ejether** [comment]
Ready for review.

File: reckoner/chart.py [hunk 2]
```diff
@@ -295,6 +295,34 @@ def template(self, default_namespace=None, default_namespace_management={}, cont
         finally:
             self.clean_up_temp_files()
 
+    def get_manifest(self, default_namespace=None, default_namespace_management={}, context=None) -> None:
+        self.__pre_command(default_namespace, default_namespace_management, context)
+        try:
+            # get_manifest needs a differnt set of args
```
Comment (sudermanjr) [inline reckoner/chart.py:301]:
```suggestion
            # get_manifest needs a different set of args
```

**Comment by ejether** [comment]
I think, the only current scenario where that provider change would bite us is if, when we implement another helm command, the person doing the code adds extra spaces between the command terms. 
`get    manifests`  would end up like `['get',' ',' ','manifests']` in the args.

Seems unlikely

**Pull request merged**